collectors/hra/parser.py

```diff
@@ -57,6 +57,11 @@ def parse_record(url, item):
     data['other_comments'] = item['OtherComments']
     data['research_summary_suitability'] = item['ResearchSummarySuitability']
 
+    # Ignore deferred records
+    date_fields = ['publication_date', 'decision_date', 'updated_date']
```

Comment:
To me `Publication of this data is currently deferred.` sounds like we should check only `publication_date` but I couldn't make requests to HRA to investigate this further so I just check all date fields because they will [raise at conversion](https://github.com/opentrials/collectors/blob/master/collectors/base/record.py#L75) if they have this deferred message.